traffic_portal/app/src/common/directives/treeSelect/tree.select.tpl.html

```diff
@@ -0,0 +1,40 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<div class="tree-select-root">
+    <input type="text" id="{{ handle }}" name="{{ handle }}" class="form-control display-field" ng-model="selected.label" ng-click="toggle()" required readonly>
+    <div class="tree-drop-down" ng-show="shown">
+        <div class="input-group search-box">
+            <label class="input-group-addon has-tooltip control-label" for="searchText">
```

Comment:
`for` must be the `id` of the labeled element - right now the `input` has a `name` but no `id`.

Also as IDs must be unique within a document, I'd recommend using the `handle` input to provide part of the ID, since that must already be unique above.